projects/imbu/engine/fluent_api.py

```diff
@@ -35,23 +35,30 @@
 
 
 
-# There is no longer training in imbu web app.  User must specify loadPath
+# No training in Imbu web app, user must specify loadPath
 if "IMBU_LOAD_PATH_PREFIX" in os.environ:
   _IMBU_LOAD_PATH_PREFIX = os.environ["IMBU_LOAD_PATH_PREFIX"]
 else:
   raise KeyError("Required IMBU_LOAD_PATH_PREFIX missing from environment")
 
-g_models = {} # Global model cache
-
-imbu = ImbuModels(
-  cacheRoot=os.environ.get("MODEL_CACHE_DIR", os.getcwd()),
-  modelSimilarityMetric=None,
-  dataPath=os.environ.get("IMBU_DATA",
-                          pkg_resources.resource_filename(__name__,
-                                                          "data.csv")),
-  retina=os.environ["IMBU_RETINA_ID"],
-  apiKey=os.environ["CORTICAL_API_KEY"]
-)
+g_imbus = {}  # Global ImbuModels cache
+g_models = {}  # Global NLP model cache
+for datasetName in os.listdir(_IMBU_LOAD_PATH_PREFIX):
+  datasetPath = os.path.join(_IMBU_LOAD_PATH_PREFIX, datasetName)
+  if os.path.isdir(datasetPath):
+    # Create an imbu instance for each dataset
+    g_imbus.update(datasetName=ImbuModels(
```

Comment:
@BoltzmannBrain It should be `g_imbus[datasetName] = ImbuModels(...`

Otherwise you will end up with a dictionary having the string 'datasetName' as key instead of the value for `datasetName` as key

Try this out:
```python
g_imbus = {}
datasetName = 'sample_review'
g_imbus.update(datasetName='blah')  # {'datasetName': 'blah'}
g_imbus['sample_review']  # throws KeyError: 'sample_review'
```